addons/website_event_track/models/event_track.py

```diff
@@ -630,3 +648,91 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
         )
 
         return track_candidates[:limit]
+
+    def _get_reminder_times_warning(self):
+        return Markup('<strong><u>%(warning_title)s</u></strong>: %(warning_content)s') % {
+            'warning_title': _('Note'),
+            'warning_content': _(
+                'The start and end times of the talk were not specified when you asked to add them to your calendar, '
+                'therefore the times indicated in this reminder correspond to those of the event.'
+            )
+        } if not self.date else ''
+
+    def _get_external_description(self):
+        """ Adding the URL of the event track into the description """
+        self.ensure_one()
+        return Markup("<a href='%(event_track_url)s'>%(name)s</a>\n%(shorten_description)s\n\n%(reminder_times_warning)s") % {
+            'event_track_url': werkzeug.urls.url_join(self.get_base_url(), self.website_url),
+            'name': self.name,
+            'shorten_description': shorten(html_to_inner_content(self.description), 1900),
+            'reminder_times_warning': self._get_reminder_times_warning(),
+        }
+
+
+    def _get_event_track_reminder_dates(self):
+        return {
+            'date_begin': self.date or self.event_id.date_begin,
+            'date_end': self.date_end or self.event_id.date_end,
+        }
+
+    def _get_yahoo_resource_url(self):
+        date_tz = self.event_id.date_tz
+        reminder_dates = self._get_event_track_reminder_dates()
+
+        params = {
+            'title':  f'{self.event_id.name}: {self.name}',
+            'in_loc': self.location_id.name,
+            'st': reminder_dates['date_begin'].astimezone(timezone(date_tz)).strftime('%Y%m%dT%H%M%S'),
+            'et': reminder_dates['date_end'].astimezone(timezone(date_tz)).strftime('%Y%m%dT%H%M%S'),
+            'desc': shorten(html_to_inner_content(self.description), 1900) + f'\n\n{html_to_inner_content(self._get_reminder_times_warning())}'
+        }
+        return YAHOO_CALENDAR_URL + werkzeug.urls.url_encode(params)
+
+    def _get_event_track_resource_urls(self):
```

Comment:
:socks: resource is a bit weird in my opinion (not being an expert at those things). Maybe
```suggestion
    def _get_track_calendar_urls(self):
```